pkgs/top-level/python-packages.nix

```diff
@@ -4196,6 +4196,28 @@ in modules // {
       description = "pytest plugin with mechanisms for caching across test runs";
     };
   };
+  
+  pytestdjango = buildPythonPackage rec {
+    name = "pytest-django-${version}";
+    version = "2.9.1";
+
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/p/pytest-django/${name}.tar.gz";
+      sha256 = "1mmc7zsz3dlhs6sx4sppkj1vgshabi362r1a8b8wpj1qfximpqcb";
+    };
+    
+    # doing this to allow depending packages to find
+    # pytest's binaries
+    pytest = self.pytest;
```

Comment:
This particular case is to allow a depending package to directly reference the `py.test` binary so it can create a wrapper - it just seemed slightly neater for the depending package to only require pytestdjango as an argument rather than pytest _and_ pytestdjango, while having the bonus of ensuring there isn't any version mixup.